### ISSUE TITLE
Tag-110- ReportWriter-Values in the report are not comparable to the values on the screen. 

### ISSUE THREAD

**Issue by rmehri** [NCIEVS/nci-protege5#130]
Values in the report are not comparable to the values on the screen.  Also “Test It” is listed once in the report but twice in the app.   Please see attached zip file to play the video for more details.

*Note: The data not being the data on the screen and one having two entries in the app but only one in the report. 

Thanks,
Reza 

[Report Issue.zip](https://github.com/NCIEVS/nci-protege5/files/651881/Report.Issue.zip)

**Comment by bdionne** [comment]
@rmehri --

Would you please just enumerate the steps to reproduce the error. It doesn't have to be detailed, just the main steps. Please also state which ontology you use to see the error and attach any relevant input or output files.

The videos aren't particularly helpful and they are inefficient to view.

Thanks,

Bob

**Comment by rmehri** [comment]
Hi Bob, 
Please see attached steps  to reproduce and attached zip file for report writer log.  Please let me know if you need more information.  I hope that helps. 

Thanks,
Reza

[Steps to reproduce.docx](https://github.com/NCIEVS/nci-protege5/files/652899/Steps.to.reproduce.docx)

[Reza_report.zip](https://github.com/NCIEVS/nci-protege5/files/652903/Reza_report.zip)

**Comment by bdionne** [comment]
@rmehri -- thanks, IO'll look into it. One of the nice features of Github is that you can type directly into these comment fields, which makes it easier for other readers

````
Steps to reproduce:
Log into this server:  https://nciws-d699-v.nci.nih.gov:9090
User eric user
Select NCI Edit Tab 
Select Anatomic Structure, System, or Substance/Body Cavity/BDA Test
Select Edit tab/Complex Properties tab and display contents
Select Report Writer and click Configure link
In Report Type click Ok and generate a report writer 
Open the Report Writer and compare it with Edit tab /properties contents
You will see the following differences:
  Test Results:
FuLL_SYN:   BDA Test

Term Type in application, but in report writer = Term Group  
Term Type = PT in application, but in report writer = Term Group = SY 
Source-code = “NADA” in the application, but in report writer, Source-Code = blank 
Sub source name = “Test” in the application, but in report writer, Sub source name = blank 

FuLL_SYN:   Barry Adding Value
Term Type in application, but in report writer = Term Group  

FuLL_SYN:   Test 1

Term Type = SN in application, but in report writer = Term Group = SY 
Term Source = PI-RADS in application, but in report writer = Term Source = NCI 
Source-code = “1ABC” in the application, but in report writer, Source-Code = blank 
Sub source name = “QA” in the application, but in report writer, Sub source name = blank 

FuLL_SYN:   Test It

Term Type = FB in application, but in report writer = Term Group = SY 
Term Source = NCI-GLOSS in application, but in report writer = Term Source = NCI
Source-code = “1234345345345353” in the application, but in report writer, Source-Code = blank 
Sub source name = “qwerty” in the application, but in report writer, Sub source name = blank 

FuLL_SYN:   Test It (duplicate name) Missing from ReportWriter, but in application is as follows:

Term Type = HD, term source= NCI-Gloss, Source code= blank, sub source= blank.
FuLL_SYN:   BDA Child Class

Term Type = SY in application, but in report writer = Term Group = SY 
Term Source = NCI in application, but in report writer = Term Source = NCI

````

**Comment by rmehri** [comment]
As per development's comment, this issue is under investigation.

**Comment by rmehri** [comment]
Hi, 
I have retested this fix using tag-118, edittab-03.4. Verified that the same issues exist.  Please see attached zip file for screen shots and reports.  Please let me know if you need more information.

[report-130 R.zip](https://github.com/NCIEVS/nci-protege5/files/748916/report-130.R.zip)

Thanks,
Reza

**Comment by rmehri** [comment]
Hi, 
This fix has been retested and verified using tag-121, nci-edittab-04.4.  Please see attached test results for more details.

[Uploading lucene-export2.txt…]()
[Uploading ReportWriter-Test Results.docx…]()
Thanks,
Reza
